features/install.feature

```diff
@@ -78,6 +78,29 @@ Feature: Installation
       And the file "config/initializers/localeapp.rb" should contain "config.api_key = ENV['LOCALEAPP_API_KEY']"
       And the exit status should be 0
 
+  Scenario: Running install with .env
+    In order to configure my non rails project and check my api key is correct
+    Given I have a valid project on localeapp.com with api key "MYAPIKEY"
+    And a file named ".env" with ""
```

Comment:
Never more used, as the behaviour is now directed solely by command line options, regardless of the `.env` file's existence.